src/twisted/trial/_synctest.py

```diff
@@ -94,15 +102,50 @@ def makeTodo(value):
 
     @return: A L{Todo} object.
     """
+    if isinstance(value, Todo):
+        return value
     if isinstance(value, str):
         return Todo(reason=value)
     if isinstance(value, tuple):
         errors, reason = value
         try:
-            errors = list(errors)
+            errors = list(errors)  # type: ignore[arg-type]
         except TypeError:
-            errors = [errors]
+            errors = [errors]  # type: ignore[list-item]
         return Todo(reason=reason, errors=errors)
+    return None
+
+
+_TCallable = TypeVar("_TCallable", bound=Callable[..., object])
+_DecoratorType = Callable[[_TCallable], _TCallable]
+_T = TypeVar("_T")
+
+
+def _noop(v: _T) -> _T:
+    return v
+
+
+def mark(
```

Comment:
based on https://github.com/escaped/django-admin-display

Reply:
This looks good... but it would be nice to have a docstring for it.